cassandra/cassandra-docker.sh

```diff
@@ -55,17 +55,23 @@ do
     --keystore_password_file=*)
       KEYSTORE_PASSWORD_FILE="${args#*=}"
     ;;
-    --truststore_file=*)
-      TRUSTSTORE_FILE="${args#*=}"
+    --truststore_nodes_file=* | --truststore_file=*)
+      TRUSTSTORE_NODES_FILE="${args#*=}"
     ;;
-    --truststore_password=* | --trustsotre_password=*)
-      TRUSTSTORE_PASSWORD="${args#*=}"
+    --truststore_nodes_password=* | --truststore_password=* | --trustsotre_password=*)
+      TRUSTSTORE_NODES_PASSWORD="${args#*=}"
     ;;
-    --truststore_password_file=*)
-      TRUSTSTORE_PASSWORD_FILE="${args#*=}"
+    --truststore_nodes_password_file=* | --truststore_password_file=*)
+      TRUSTSTORE_NODES_PASSWORD_FILE="${args#*=}"
     ;;
-    --cassandra_pem_file=*)
-      CASSANDRA_PEM_FILE="${args#*=}"
+    --truststore_client_file=* | --truststore_file=*)
+      TRUSTSTORE_CLIENT_FILE="${args#*=}"
+    ;;
+    --truststore_client_password=* | --trustsotre_password=*)
+      TRUSTSTORE_CLIENT_PASSWORD="${args#*=}"
+    ;;
+    --truststore_client_password_file=* | --truststore_password_file=*)
+      TRUSTSTORE_CLIENT_PASSWORD_FILE="${args#*=}"
```

Comment:
Alright, I'll remove the following options in an individual commit:

* truststore_client_password_file
* truststore_client_password
* truststore_client_file
* truststore_nodes_password_file
* truststore_nodes_password
* truststore_nodes_file
* keystore_password_file
* keystore_password
* keystore_file

> We should probably be allowing a user to pass a root CA bundle then instead of an individual CA

That's the current situation on this latest commit: each trust store is built based on all certs on a bundle (truststore_nodes_authorities and truststore_client_authorities)